code/controllers/subsystem/processing/fishing.dm

```diff
@@ -1,7 +1,4 @@
-/**
- * So far, only used by the fishing minigame. Feel free to rename it to something like veryfastprocess
- * if you need one that fires 10 times a second
- */
+/// subsystem for the fishing minigame processing.
 PROCESSING_SUBSYSTEM_DEF(fishing)
 	name = "Fishing"
-	wait = 0.1 SECONDS
+	wait = 0.05 SECONDS // If you raise it to 0.1 SECONDS, you better also modify [datum/fish_movement/move_fish()]
```

Comment:
we sure about this one? i'm on board but I'm worried it'll have trouble keeping up on live